verilog/CST/data.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2017-2019 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This unit provides helper functions that pertain to SystemVerilog
+// net declaration nodes in the parser-generated concrete syntax tree.
+
+#ifndef VERIBLE_VERILOG_CST_DATA_H_
+#define VERIBLE_VERILOG_CST_DATA_H_
+
+#include <vector>
+
+#include "common/analysis/syntax_tree_search.h"
+#include "common/text/concrete_syntax_leaf.h"
+#include "common/text/concrete_syntax_tree.h"
+#include "common/text/symbol.h"
+
+namespace verilog {
+
+using verible::Symbol;
+using verible::SyntaxTreeLeaf;
+
+std::vector<const SyntaxTreeLeaf*> GetIdentifiersFromDataDeclaration(
```

Comment:
As this is a public function, add a comment like:
"Returns tree leaves that correspond to declared names in data declarations."
If you like, this could take the addition step of calling `.get()` on each leave to get the underlying `const TokenInfo&`, since the leaf doesn't contain more information than TokenInfo.

Reply:
That makes sense - added the comments and changed the returned value, thanks!